pecos/xmc/xlinear/model.py

```diff
@@ -133,6 +133,23 @@ def load(cls, model_folder, is_predict_only=False, **kwargs):
         )
         return cls(model)
 
+    @classmethod
+    def compile_mmap_model(cls, npz_folder, mmap_folder):
+        """
+        Compile xlinear model from npz format to memory-mapped format
+        for faster loading and referencing.
+        Args:
+            npz_folder (str): The source folder path for xlinear npz model.
+            mmap_folder (str): The destination folder path for xlinear mmap model.
+        """
+        import shutil
+
+        shutil.copyfile(path.join(npz_folder, "param.json"), path.join(mmap_folder, "param.json"))
```

Comment:
Should we add `"compiled_format": "memory_map"` in the `params.json` so that user need not to know the format before calling `load`?